content/docs/tutorials/google-public-ca/README.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Configuring Google's Public Certificate Authority"
+linktitle: "Configuring Google's Public Certificate Authority"
+---
+
+For years we used `cert-manager` to provision TLS certificates from ZeroSSL. Their ACME service is free, but we've really gotten what we paid for. Service outages were common, and more recently ZeroSSL added undocumented rate limiting for HTTP requests to their ACME API. This change put us in the same situation as this `cert-manager` user: <https://github.com/cert-manager/cert-manager/issues/5867>. We began looking for alternative certificate authorities that support the ACME protocol.
```

Comment:
"We" isn't defined here, but we can rephrase it so that "we" isn't needed.

```suggestion
cert-manager can be used for issuing free certificates with ZeroSSL and Let's Encrypt. 

ZeroSSL's is great to get started but doesn't work well for large scales use-cases. More specifically, ZeroSSL has a rate limit for HTTP requests to their ACME API.

For companies that need to go beyond ZeroSSL's and Let's Encrypt's limitations, Google's Public Certificate Authority is a good fit.
```